pkg/virt-launcher/virtwrap/cmd-server/server_test.go

```diff
@@ -333,19 +324,19 @@ var _ = Describe("Virt remote commands", func() {
 			It("does not return exit code errors", func() {
 				expectExec().Times(1).Return("", agent.ExecExitCode{ExitCode: 1})
 				_, err := server.Exec(context.TODO(), execRequest())
-				Expect(err).To(BeNil())
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
I would advise being consistent. Either you are using `ToNot(HaveOccurred)` or `To(Succeed())`. Did I miss anything?

Reply:
Asserting an error directly reads correct with !HaveOccured(), because it talks about the error.
Succeed() fits when asserting the execution.
Foo() succeeds, but err have not occurred.

Reply:
Thanks, that make sense. Why can't we do `_, err := server.Exec(context.TODO(), execRequest())`

Reply:
Could you elaborate? That is the prev line, no?

Reply:
Never mind I now see it returns two values.

Reply:
nit: I would use `NotTo` rather than `ToNot`. Not sure if that breaks the consistency with the rest of the pkg.

Same for the next calls.